filament/backend/src/vulkan/VulkanContext.cpp

```diff
@@ -150,7 +150,7 @@ void selectPhysicalDevice(VulkanContext& context) {
 
 void createVirtualDevice(VulkanContext& context) {
     VkDeviceQueueCreateInfo deviceQueueCreateInfo[1] = {};
-    static const float queuePriority[] = {1.0f};
+    static constexpr float queuePriority[] = {1.0f};
```

Comment:
I wouldn't make this static, I would just do `float queuePriority = 1.0;` and use `&queuePriority`.